src/DotNetty.Common/ResourceLeakDetector.cs

```diff
@@ -14,17 +14,16 @@ namespace DotNetty.Common
     using DotNetty.Common.Internal;
     using DotNetty.Common.Internal.Logging;
     using DotNetty.Common.Utilities;
-    using Nito;
 
-    public sealed class ResourceLeakDetector
```

Comment:
why not seal it?

Reply:
It is not final in netty but nothing extends it for now as far as I know of.